libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -55,86 +46,62 @@ <h1>{{ transloco('title') }}</h1>
         />
       </watt-form-field>
 
-      <watt-form-field size="normal">
-        <watt-label class="watt-text-m">{{ transloco('role') }}</watt-label>
-
-        <select
-          #searchInputRole
-          wattInput
-          [(ngModel)]="this.searchCriteria.senderRoleType"
-          name="role"
-        >
-          <option selected [ngValue]="null">
-            {{ transloco('ChooseRole') }}
-          </option>
-          <option
-            *ngFor="let roleType of this.roleTypes | keyvalue"
-            [ngValue]="roleType['key']"
-            value="{{ roleType['value'] }}"
-          >
-            {{ roleType['key'] + ' - ' + roleType['value'] }}
-          </option>
-        </select>
-      </watt-form-field>
-
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('messageId')
+          transloco("receiverId")
         }}</watt-label>
 
         <input
-          #searchInputMessageId
-          [(ngModel)]="this.searchCriteria.messageId"
+          #searchInputReceiverId
+          [(ngModel)]="this.searchCriteria.receiverId"
           wattInput
           type="text"
-          name="messageId"
+          name="receiverId"
         />
       </watt-form-field>
 
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('receiverId')
+          transloco("messageId")
         }}</watt-label>
 
         <input
-          #searchInputReceiverId
-          [(ngModel)]="this.searchCriteria.receiverId"
+          #searchInputMessageId
+          [(ngModel)]="this.searchCriteria.messageId"
           wattInput
           type="text"
-          name="receiverId"
+          name="messageId"
         />
       </watt-form-field>
 
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('businessReason')
+          transloco("businessReason")
         }}</watt-label>
+        <watt-dropdown
+          [(ngModel)]="this.searchCriteria.processType"
+          [formControl]="processTypeFormControl"
```

Comment:
issue: it's not recommended to use `ngModel` and `formControl` on the same field. The `watt-dropdown` component works with either, so you can remove one of them. By looking at the file as a whole, you're mostly using the template-driven approach, so you can delete the `formControl`. Both here and the `processTypeFormControl` property.